ts/session/sending/MessageQueueInterface.ts

```diff
@@ -19,8 +20,5 @@ export interface MessageQueueInterface {
   sendUsingMultiDevice(user: PubKey, message: ContentMessage): void;
   send(device: PubKey, message: ContentMessage): void;
   sendToGroup(message: GroupMessageType): void;
-  sendSyncMessage(
-    message: ContentMessage,
-    sendTo: Array<PubKey>
-  ): Promise<Array<void>>;
+  sendSyncMessage(message: SyncMessage | undefined): Promise<any>;
```

Comment:
Since we need to convert to SyncMessage every time we call, why not just take an optional ContentMessage?

Reply:
For type safety. We want `sendSyncMessage` to only be called for a sync message. There are a few of these messages that can't be converted from existing ones, these are your GroupSync, ContactSync, etc..